app/js/sendToChart.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * function to generate the data object for the chart displayed on the front end.
+ * @returns the data object for graph.js
+ */
+async function sendToChart() { 
+    let users = await createUsersObject();
+    users = searchAndFilter(users.data);
```

Comment:
Just out of curiousity, is there a reason you are making another AJAX request here and filtering it? As far as I can tell this will result in the exact same data that is already being requested and fitlered in the updateScoreTable function. handlebarsScoreTable.js line 26. the filteredUserArray variable should contain the same information and request everytime a filter/search is applied.

Reply:
I guess we could hook updateGraph into updateScoreTable, but we weren't sure whether you would change the functionality of updateScoreTable as part of your pagination story and we figured it was better to have our function able to work independently.

Reply:
Also keep in mind that when that code was written, searchAndFilter was still using its silly implementation which wouldn't have allowed that. We'll refactor on Ash's suggestion in Code Review.

Reply:
Change made on Ash's suggestion